### PR TITLE
Add ⚡️ as default favicon for documentation and examples (#191)

### PR DESCRIPTION
**Pull request by NayanVR**
#191 

Added ⚡️ icon as svg favicon to documentation pages and all examples repos.

### CODE REVIEW

File: apps/documentation/src/routes/__root.tsx [hunk 1]
```diff
@@ -73,18 +73,27 @@ export default function RootRoute({ children }: RootRouteProps): JSX.Element {
...
-          sizes="32x32"
-          href="/favicon-32x32.png"
-        />
-        <link
-          rel="icon"
-          type="image/png"
-          sizes="16x16"
-          href="/favicon-16x16.png"
-        />
+        {pathname.startsWith('/documentation') ? (
```
Comment (Valerioageno) [inline apps/documentation/src/routes/__root.tsx:76]: Could you please apply this change to all the pages and remove the not used any more files from the `public/` folder?

File: examples/tuono-app/src/routes/__root.tsx [hunk 2]
```diff
@@ -1,9 +1,20 @@
...
 
 export default function RootRoute({ children }: RootRouteProps): JSX.Element {
-  return <main className="main">{children}</main>
+  return (
+    <>
+      <Head>
+        <link
+          rel="icon"
+          href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>⚡</text></svg>"
+        />
```
Comment (Valerioageno) [inline examples/tuono-app/src/routes/__root.tsx:15]:
```suggestion
        />
        <title>Tuono app</title>
```

File: examples/with-mdx/src/routes/__root.tsx [hunk 3]
```diff
@@ -1,14 +1,23 @@
...
   return (
-    <main className="main">
-      <MDXProvider components={{}}>{children}</MDXProvider>
-    </main>
+    <>
+      <Head>
+        <link
+          rel="icon"
+          href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>⚡</text></svg>"
+        />
```
Comment (Valerioageno) [inline examples/with-mdx/src/routes/__root.tsx:16]:
```suggestion
        />
        <title>Tuono - With MDX</title>
```

**Review by Valerioageno — CHANGES_REQUESTED** [review]
I left some minor comments. LGTM Thanks

**Review by NayanVR — COMMENTED** [review]
I just amend the commit. Waiting for your feedback

File: apps/documentation/src/routes/__root.tsx (hunk 1 above)
Comment (Valerioageno) [reply apps/documentation/src/routes/__root.tsx:76]: @NayanVR you should remove this condition and apply the new favicon to all the pages

Comment (NayanVR) [reply apps/documentation/src/routes/__root.tsx:76]: Okay got it. Should I apply new favicon to hero page (landing page) also?

Comment (Valerioageno) [reply apps/documentation/src/routes/__root.tsx:76]: Yes

Comment (NayanVR) [reply apps/documentation/src/routes/__root.tsx]: Updated the commit

**Comment by Valerioageno** [comment]
Consider rebasing since we have just merged this https://github.com/tuono-labs/tuono/pull/187

**Comment by NayanVR** [comment]
> Consider rebasing since we have just merged this #187

Rebased the same commit

**Review by Valerioageno — APPROVED** [review]
LGTM. Thanks

**Pull request merged**